lib/game_logic.rb

```diff
@@ -0,0 +1,89 @@
+class TicTacToe
+
+  def initialize
+    welcome
```

Comment:
A more typical pattern here would be to simply call `TicTacToe.new.welcome` in the bin file.